### ISSUE TITLE
Hidden room list scrollbars in redesign causes confusion

### ISSUE THREAD

**Issue by non-Jedi** [vector-im/riot-web#8400]
- **Browser**: qutebrowser
- **OS**: linux

### Description

On riot.im/develop, since the redesign, there's no visual indication of how far you've scrolled down in any room list (no scrollbar unless hovering). This is confusing since rooms can disappear by moving to the top of the room list from new activity and because it can hide rooms that have notifications (pinned to top of list) without any indication that user is currently scrolled down in list.

Basically the problem is that the default position where a list is scrolled to the top and the position where the list is scrolled down some can be visually identical. This is problematic since position of a room in the list carries meaning for the app.

**Comment by ara4n** [comment]
you should be able to spot when you're scrolled down as there's a fade-out effect applied to the top of the section. is this not happening? or are you aligning the scroll such that the fade-out is invisible?

**Comment by non-Jedi** [comment]
The latter. If the scroll is aligned so that no room is half-visible (fairly easy to do either on purpose or accidentally), the fade-out is not visible.

**Comment by turt2live** [comment]
Fixed in the new room list
